subsys/zigbee/lib/zigbee_shell/src/zigbee_shell_utils.c

```diff
@@ -38,9 +38,9 @@ zb_uint8_t zb_shell_ep_handler(zb_bufid_t bufid)
 	uint8_t ep_handler_cnt = (sizeof(zb_ep_handlers) /
 				  sizeof(zb_device_handler_t));
 
-#if defined(CONFIG_ZIGBEE_SHELL_DEBUG_CMD) && defined(CONFIG_LOG)
-	(void)(zigbee_logger_eprxzcl_ep_handler(bufid));
-#endif
+	if (IS_ENABLED(CONFIG_ZIGBEE_LOGGER_EP)) {
```

Comment:
Now that I think of it the Zigbee logger endpoint really should have dependency to LOG